po/de.po

```diff
@@ -15396,6 +15396,7 @@ msgstr "Ungültiger Sektionsname: %s"
 #, c-format
 msgid "refusing to work with overly long line in '%s' on line %<PRIuMAX>"
 msgstr ""
+"Ausführung mit übermäßig langer Zele in ‚%s‘ bei Zeile %<PRIuMAX> verweigert"
```

Comment:
Zele = Zeile ?
and it should be `'%s'` instead?

Reply:
fixed